calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -79,6 +79,34 @@ To change default settings, modify the `imageassurance` installation resource.
 
     Set the `containerdVolumeMountPath`. Default: `/var/lib/containerd/`.
 
+### Configure exclusions for image scanning
+
+To specify which namespaces should be excluded from future scans, follow the following steps.
+
+- Modify your [Image Assurance](../reference/installation/ia-api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec) installation resource to include the `exclusions.namespaces` field. List each namespace you want to exclude.
+
+
+```yaml
+apiVersion: image-assurance.operator.tigera.io/v1
+kind: ImageAssurance
+metadata:
+  name: default
+spec:
+  clusterScanner: Enabled
+  exclusions:
+    namespaces:
+      - "kube-system"
+      - "dev-qa"
+```
+
+In this example, the workloads in the `kube-system` and `dev-qa` namespaces are excluded from future image scans. 
+
+:::note
+
+Applying or updating namespace exclusions affects only future scans. Results from scans conducted prior to these exclusions will remain unchanged. The exclusions configured will apply to both cluster scanner and runtime view (**Running Images** tab).
+
```

Comment:
May be worth being extremely detailed and say:

```
Deleting an excluded namespace from the ImageAssurance Installation (making it included again) will make the cluster-scanner scan the images in that namespace right away.
```

Kind of implicit, but might be worth being super clear.